tutordistro/patches/openedx-lms-production-settings

```diff
@@ -1,26 +1,32 @@
 {% for pkg in iter_values_named(suffix="_DPKG") %}
-{%- if pkg != 'None' -%}
+{%- if pkg != 'None' and pkg["variables"] is defined %}
 # Start {{ pkg["name"] }} settings
 {%- if "variables" in pkg and "production" in pkg["variables"] -%}
 {%- for key, value in pkg["variables"]["production"].items() %}
-{{ key }} = {% if value is string %}"{{ value }}"{% else %}{{ value }}{% endif %} 
+{{ key }} = {% if value is string %}"{{ value }}"{% else %}{{ value }}{% endif %}
 {%- endfor %}
 # End {{ pkg["name"] }} settings
 {%- endif -%}
 {%- endif -%}
 {% endfor %}
 
+{% if DISTRO_THEME_DIRS is defined -%}
 COMPREHENSIVE_THEME_DIRS.remove('/openedx/themes')
 COMPREHENSIVE_THEME_DIRS.extend({{ DISTRO_THEME_DIRS }})
+{%- endif %}
 
+{%- for pkg in iter_values_named(suffix="_DPKG") %}
+{%- if pkg["name"] == 'eox-theming' %}
 ################## EOX_THEMING ##################
```

Comment:
Leaving this is not as leave the default? If someone wants to install eox-theming if not their responsibility to add this config somewhere?

Reply:
If user want to use eox-theming this is the basic configuration, and we have some cases at the moment:
1. We could remove this setting and leave the responsibility to users, but remember that we own both products, it's our responsibility to these work as expected. 
2. We could add some additional feature to add this setting in config.yml, but as I see it, each time this file becomes heavier.
3. With this configuration we make sure that eox-theming is used well if the user wants to use it, otherwise the configuration is simply not added, and in case the plugin has many changes that break the current behavior, we could deprecate this configuration. In the Changelog, I have not seen changes that break the current functionality in the last few releases.

At this point I think we can continue to work with option three, it is not a change that affects the goal of removing the default settings, because it is simply a consequence of using eox-theming and it is optional.